### PR TITLE
Search: initial changelog

### PR DESCRIPTION
**Pull request by kraftbj**
- Changelog and readme
- Changelog to Readme
- screenshots
- Contributors

Initial release.

#### Changes proposed in this Pull Request:

* Changelog release.

#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Read.

### CODE REVIEW

File: projects/plugins/search/readme.txt [hunk 1]
```diff
@@ -12,57 +12,109 @@ A cloud-powered replacement for WordPress' search.
...
+
+You can purchase a Search subscription directly through this plugin or via the [Jetpack website](https://jetpack.com/search/)
+
+== Screenshots ==
+
+1. Settings interface
+2. Overlay customization
+3. Overlay interface
+4. Overlay details: filtering and spelling correction
+5. My Jetpack
```
Comment (jsnmoon) [inline projects/plugins/search/readme.txt:110]:
```suggestion
1. Configuring and customizing Jetpack Search is a breeze.
2. Lots of customization options to make Search match your site design.
3. Clear, easy-to-read search results.
4. Easy to filter results and smart spelling mistake handling.
5. Manage all of your Jetpack products, including Search, in a single place.
```

File: projects/js-packages/connection/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
 
 ### This is a list detailing changes for the Jetpack RNA Connection Component releases.
 
+## 0.18.3 - 2022-05-30
+### Added
+- Add option to use the checkout workflow from the ConnectionScreenRequirePlan component
+
+### Changed
+- Updated package dependencies
+- Updated package dependencies.
```
Comment (anomiex) [inline projects/js-packages/connection/CHANGELOG.md:11]:
```suggestion
- Updated package dependencies
```

File: projects/plugins/search/readme.txt [hunk 3]
```diff
@@ -12,57 +12,109 @@ A cloud-powered replacement for WordPress' search.
...
-Ordered list:
+Yes, Jetpack Search is optimized for site visitors from mobile devices.
 
-1. Some feature
-1. Another feature
-1. Something else about the plugin
+= Will this plugin work with my theme? =
 
-Unordered list:
+This plugin works seamlessly with any WordPres theme.
```
Comment (bluefuton) [inline projects/plugins/search/readme.txt:94]:
```suggestion
This plugin works seamlessly with any WordPress theme.
```

File: projects/js-packages/idc/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,11 @@
 
 ### This is a list detailing changes for the Jetpack RNA IDC package releases.
 
+## 0.10.12 - 2022-05-30
+### Changed
+- Updated package dependencies
+- Updated package dependencies.
```
Comment (anomiex) [inline projects/js-packages/idc/CHANGELOG.md:8]:
```suggestion
- Updated package dependencies
```

File: projects/js-packages/licensing/CHANGELOG.md [hunk 5]
```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.5.2 - 2022-05-30
+### Changed
+- Updated package dependencies
+- Updated package dependencies.
```
Comment (anomiex) [inline projects/js-packages/licensing/CHANGELOG.md:11]:
```suggestion
- Updated package dependencies
```

File: projects/packages/my-jetpack/CHANGELOG.md [hunk 6]
```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
...
 
+## [1.6.0] - 2022-05-30
+### Added
+- Added Social to My Jetpack.
+
+### Changed
+- Make My Jetpack use the new Modules class
+- Replace deprecated external-link variation by using isExternalLink prop
+- Updated package dependencies
+- Updated package dependencies.
```
Comment (anomiex) [inline projects/packages/my-jetpack/CHANGELOG.md:16]:
```suggestion
- Updated package dependencies
```

**Review by anomiex — APPROVED** [review]

**Comment by jeherve** [comment]
r246583-wpcom

**Pull request merged**